crates/pop-parachains/src/new_parachain.rs

```diff
@@ -39,37 +35,50 @@ pub fn instantiate_standard_template(
 	config: Config,
 	tag_version: Option<String>,
 ) -> Result<Option<String>> {
-	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
-	let source = temp_dir.path();
+	// Template palceholder definitions
+	let mut token_symbol: String = "token-symbol=".to_string();
+	let mut token_decimals: String = "token-decimals=".to_string();
+	let mut initial_endowement: String = "initial-endowment=".to_string();
 
-	let tag = Git::clone_and_degit(template.repository_url()?, source, tag_version)?;
+	// Placeholder customization
+	token_symbol.push_str(&*config.symbol);
+	token_decimals.push_str(&*config.decimals);
+	initial_endowement.push_str(&*config.initial_endowment);
 
-	for entry in WalkDir::new(&source) {
-		let entry = entry?;
+	// Tempalte rendering arguments
+	let standard_template_path = TemplatePath {
+		git: Some(
+			template
+				.repository_url()
+				.map_or(String::from("https://github.com/r0gue.io/base-parachain"), |r| {
```

Comment:
The `template.repository_url()` should never produce an error because it is our responsibility to set the repository URL correctly.
Not sure the best approach here, personally I'd throw an error instead of hardcoding the `base-parachain` url, or if we don't want to throw an error return a the DEFAULT value in the `repository_url` function(https://github.com/r0gue-io/pop-cli/blob/main/crates/pop-parachains/src/templates.rs#L225, )